src/site/content/en/blog/debug-web-vitals-in-the-field/index.md

```diff
@@ -277,157 +285,175 @@ function wasFIDBeforeDCL(fidEntry) {
   entry for that script.
 {% endAside %}
 
-#### Identify the FID target element
+#### Identify the FID target element and event type
 
-Another potentially useful debug signal is the element that was interacted with.
-While the interaction with the element itself does not contribute to FID
-(remember FID is just the delay portion of the total event latency), knowing
-which elements your users are interacting with may be useful in determining how
-best to improve FID.
+Additional potentially-useful debug signals are the element that was interacted
+with as well as the type of interaction it was (such as `mousedown`, `keydown`,
+`pointerdown`). While the interaction with the element itself does not
+contribute to FID (remember FID is just the delay portion of the total event
+latency), knowing which elements your users are interacting with may be useful
+in determining how best to improve FID.
 
 For example, if the vast majority of your user's first interactions are with a
-particular element, it consider inlining the JavaScript code needed for that
+particular element, consider inlining the JavaScript code needed for that
 element in the HTML, and lazy loading the rest.
 
 To get the element associated with the first input event, you can reference the
 `first-input` entry's `target` property:
```

Comment:
```suggestion
To get the element associated with the first input event, you can reference the
`first-input` entry's `target` and `name` properties:
```

Reply:
Thanks, updated myself to also reference the name on 301.